java/Mekhmet-Efe_Adzhar/src/com/zoolatech/lecture4/tasks/_5/Number5.java

```diff
@@ -7,54 +7,35 @@
  */
 
 import java.math.BigDecimal;
-import java.util.InputMismatchException;
 import java.util.Scanner;
 
 public class Number5 {
     public static void main(String[] args) {
         Scanner scanner = new Scanner(System.in);
         System.out.println("Input first number, operation and second number");
-        Calculation calculation = new Calculation(scanner.nextBigDecimal(), scanner.next().charAt(0), scanner.nextBigDecimal());
-        System.out.println(calculation.calculations());
+        System.out.println(calculations(scanner.nextBigDecimal(), scanner.next().charAt(0), scanner.nextBigDecimal()));
     }
-}
 
-class Calculation {
-    BigDecimal numberOne;
-    char operation;
-    BigDecimal numberTwo;
-
-    public Calculation(BigDecimal numberOne, char operation, BigDecimal numberTwo) {
-        this.numberOne = numberOne;
-        this.operation = operation;
-        this.numberTwo = numberTwo;
-    }
-
-    public BigDecimal calculations() {
-        BigDecimal bigDecimal = null;
-        try {
-            switch (operation) {
-                case '+':
-                    bigDecimal = numberOne.add(numberTwo);
-                    return bigDecimal;
-                case '-':
-                    bigDecimal = numberOne.subtract(numberTwo);
-                    return bigDecimal;
-                case '*':
-                    bigDecimal = numberOne.multiply(numberTwo);
-                    return bigDecimal;
-                case '/':
-                    bigDecimal = numberOne.divide(numberTwo);
-                    return bigDecimal;
-                case '%':
-                    bigDecimal = numberOne.remainder(numberTwo);
-                    return bigDecimal;
-                default:
-                    return bigDecimal;
+    public static BigDecimal calculations(BigDecimal numberOne, char operation, BigDecimal numberTwo) {
+        switch (operation) {
```

Comment:
switch expression would allow you to remove multiple return statements:

```
return switch(operation) {
   case '+' -> numberOne.add(numberTwo);
   ...
}